tests/phpunit/TestFramework/Coverage/JUnit/JUnitTestCaseSorterTest.php

```diff
@@ -96,4 +103,133 @@ public function test_it_returns_unique_and_sorted_by_time_test_cases(): void
         $this->assertCount(3, $uniqueSortedFileNames);
         $this->assertSame('/path/to/test-file-3', $uniqueSortedFileNames[0]);
```

Comment:
Same here: maybe worth checking against the whole array, or if the values are annoying we can do:

```php
$this->assertSame(
    [
        '/path/to/test-file-1',
        '/path/to/test-file-2',
        '/path/to/test-file-3',
    ],
    array_keys($uniqueSortedFileName)
);
```